src/MarginTrading.Backend.Services/Infrastructure/FakeIdentityGenerator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Threading.Tasks;
+using MarginTrading.Backend.Core.Repositories;
+
+namespace MarginTrading.Backend.Services.Infrastructure
+{
+    public class FakeIdentityGenerator : IIdentityGenerator
+    {
+        private long _currentId;
+        
+        public FakeIdentityGenerator()
+        {
+            _currentId = DateTime.Now.Ticks;
```

Comment:
Why do you use Now instead of UtcNow?

Reply:
Because it does not matter in that case)